src/utils/did-helper.ts

```diff
@@ -22,6 +24,24 @@ export function isGenesisState(did: DID, state: bigint | string): boolean {
   return id.bigInt().toString() === idFromState.bigInt().toString();
 }
 
+/**
+ * Checks if DID is an ethereum identity
+ *
+ * @param {string} did - did
```

Comment:
string -> DID

Reply:
Yes. I have found other places with same issue and I have corrected them.